### ISSUE TITLE
Code from takosharkrules doesn't work

### ISSUE THREAD

**Issue by dskarasev** [Bendall6456/cbtags#1]
Hi! 

Carlbot can't pass the following part of the code:

{=(json):"title":"{embedtitle}","description":"{des}","fields":[{{r}(░,
):{{r}(█, ):{join(,):{{r}(┐, ):{{r}(
,░):{{r}( ,█):{cfields}}}}}}}],"author":{"name":"{server}","icon_url":"{server(icon)}"},"color":3092790}

Could you please check it

**Comment by Bendall6456** [comment]
Did you made sure you changed the rules channel ID and roles ID to your roles and rule channel?

**Comment by Bendall6456** [comment]
Hm interesting when I copied the code and changed the rules channel ID and staff role id, when I just used "-rules" , it gives me that tagscript error. However, when I did "-rules keyword" that seemed to work.

![Screenshot_20210420-040916](https://user-images.githubusercontent.com/40281058/115370203-71b44680-a18e-11eb-8414-224a037c2950.png)

**Comment by dskarasev** [comment]
Yes, I've changed all IDs but have same error messages about json Value and brackets

**Comment by Bendall6456** [comment]
Perhaps asking in the support server for [Carl-Bot](https://discord.gg/DSg744v) and see if someone helps you, I am not sure what the issue is sorry about that.

**Comment by Albert-J04** [comment]
There is an updated version there, TagScript changes on Carl-bot caused this to break.

**Comment by Maldohr** [comment]
"TagScript SyntaxError, make sure your brackets are balanced and your if statements correct!"

If I use the code. I change the channelid with my channelid but didnt work
